test/section.cooltt

```diff
@@ -0,0 +1,38 @@
+section sec1 begin
+  def x : nat := 10
+  view [ ?view ]
```

Comment:
Can you remind me what the question mark means here? I didn't see an example of it in the PR text.

Reply:
The question mark would show the current namespace. An example is in `import.cooltt`. `view` works on what's visible, so `view [ ?view ]` applies the pattern `?view` on what's visible, which ends up printing out what's visible without making any changes.

Reply:
Similarly, `repack [?label]` would inspect what's being exported and then leave it unchanged.

Reply:
Ah ok. I think part of what confused me too was that I wasn't sure if the use of the identifier `?view` was meaningfully different from `?foo` in the example. Maybe the example can be changed to show that it doesn't matter what you put there, but I obvioulsy don't feel strongly about it.

This PR looks great!